hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestResult.java

```diff
@@ -131,18 +131,6 @@ public void testCurrentOnEmptyCell() throws IOException {
     assertNull(r.current());
   }
 
-  @Test
-  public void testAdvanceTwiceOnEmptyCell() throws IOException {
```

Comment:
Better keep this test but use two assertFalse to confirm that the second advance call will return false?

Reply:
> Better keep this test but use two assertFalse to confirm that the second advance call will return false?

After the change, empty list will always return false. I think we could change the UT's name to testAdvanceMultipieOnEmptyCell and use an arbitrary value > 2 to assert false. What do you think?

Reply:
It is also fine. Just go ahead.